packages/kit/src/template.ts

```diff
@@ -27,6 +29,18 @@ export function addTemplate<T> (_template: NuxtTemplate<T> | string) {
   // Remove any existing template with the same destination path
   filterInPlace(nuxt.options.build.templates, p => normalizeTemplate(p).dst !== template.dst)
 
+  try {
+    const { source } = captureStackTrace().find(e => e.source && !e.source.startsWith('file://' + distDir)) ?? {}
```

Comment:
Since we are using url comparation, wondering if would be better to use `distDirURL` (and avoid `fileURLToPath` which is also platform dependent in utils.ts)